README.md

```diff
@@ -26,9 +26,9 @@ dotnet add package Amazon.CloudWatch.EMF
 
 ## Usage
 
-To get a metric logger, you can instantiate it like so.
+To get a metric logger, you can instantiate it using the code snippet below.
 `MetricsLogger` implements `IDisposable`. 
-When the logger is disposed, it will write the metrics to the configured sink.
+When the logger is disposed, it will write the metrics to the configured sink. Also see [Graceful Shutdown](#graceful-shutdown). You will have to call ShutdownAsync() on logger so that the program has enough time to write before shutting down. A full example can be found in the examples directory.
```

Comment:
nit: I would say that the program waits for writing to complete before shutting down

Reply:
In what situations will you "have to call ShutdownAsync() on logger" ?

Reply:
> nit: I would say that the program waits for writing to complete before shutting down

Updated!

Reply:
> In what situations will you "have to call ShutdownAsync() on logger" ?

Whenever we use out-of-process agent (the CloudWatch Agent or FireLens / Fluent-Bit) to collect the EMF events, the data gets buffered asynchronously. This means when the logger gets flushed, the buffered data may not be safely persisted. To gracefully shutdown the environment, you will have to call shutdown on the environment's sink. The  ShutdownAsync() method internally calls 'await _environment.Sink.Shutdown()' so that the program waits for writing before shutting down.